getting-started/con-oidc-overview.adoc

```diff
@@ -0,0 +1,28 @@
+[id='con-oidc-configration_{context}']
```

Comment:
In the Prerequisites section, include the role / permission required to perform this procedure.

Reply:
I see your point @linfraze - this should really be an overview of using OIDC in RHDH, so the heading should probably say that rather than using the gerund e.g. _Overview of the OIDC provider in Developer Hub_ and then rephrase the opening paragraph.

The aim of the section is to describe the OIDC/Keycloak configuration which is done in the following paragraph...so we're describing the same process twice. Or maybe we should drop this first section altogether? 

WDYT?

Reply:
I've renamed the title of this paragraph to avoid using a gerund and re-written the opening paragraph to make it a more general concept i.e. it is not specific to Keycloak which is documented in the following section.